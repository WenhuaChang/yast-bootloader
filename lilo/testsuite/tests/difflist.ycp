/*
 * Authors:
 *  Unknown <unknown@addres.xx>
 *
 * $Id$
 *
 */

{
    include "testsuite.ycp";
    import "Pkg";
    include "lilo/routines.ycp";

    TEST(``(difflist([1, 2, 3, 4], [1, 3])), [], nil);
    TEST(``(difflist([], [1, 3])), [], nil);
    TEST(``(difflist([1, 3], [1, 3])), [], nil);
    TEST(``(difflist([], [])), [], nil);
    TEST(``(difflist([1, 3], [])), [], nil);
    TEST(``(difflist([2, 2, 2], [1, 3])), [], nil);
    TEST(``(difflist([2, 2, 2], [2])), [], nil);

}
