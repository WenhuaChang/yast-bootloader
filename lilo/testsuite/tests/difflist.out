Return	[2, 4]
Return	[]
Return	[]
Return	[]
Return	[1, 3]
Return	[2, 2, 2]
Return	[]
